src/components/KExternalLink/KExternalLink.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <a
+    v-if="href"
```

Comment:
question: this logic will still render the content in the DOM if a user provides a string for `href` even if it's not valid; should we map the `v-if` to a function that checks for the validity of the href so that if an invalid string is returned, nothing renders?

If yes, you can utilize the same function in the validator that I suggested as long as you set it to a `const` before defining the props.

Reply:
Good idea!